.circleci/config.yml

```diff
@@ -181,6 +181,13 @@ jobs:
                 name: Publish plugin to Jetbrains Marketplace
                 command: ./gradlew publishPlugin
             - reconfigure_origin_remote
+            - run:
+                name: Update CHANGE-NOTES.md
+                command: |
+                  ./gradlew patchChangeLog
+                  tag=v$(extract_version_from_current_wd)
+                  git commit -a -m "Update change notes for $tag"
+                  git push
```

Comment:
Whoops no, that won't be accepted by GitHub (you can't just push commit to `master` without PR approval & CI passing first) :/

Reply:
What's more... this would cause an infinite loop of `master` builds (as each new pushed commit will trigger a new `master` build, which will then push new commit to master etc.

Reply:
Generally, pushing anything from CI to `develop`/`master` branches is very hard to get right... or rather, a massive antipattern, to be avoided at all cost